service/src/main/java/uk/gov/hmcts/reform/fpl/config/AsyncConfiguration.java

```diff
@@ -18,6 +21,9 @@
 @Configuration
 public class AsyncConfiguration implements AsyncConfigurer {
 
+    @Autowired
+    private ApplicationContext context;
```

Comment:
Do we want to have field injection or is constructor injection better?

Reply:
yeah, for consistency it will be better to change it